openfe/protocols/openmm_rbfe/equil_rbfe_methods.py

```diff
@@ -264,31 +263,16 @@ class BarostatSettings(BaseModel):
     class Config:
         arbitrary_types_allowed = True
 
-    pressure = 1 * unit.bar
     frequency = 25 * unit.timestep
```

Comment:
It might make more sense to just move this to integrator settings (renamed as `barostat_frequency`), in reality this is just a "how often should my integration loop stop and do an MC barostat move).